fineract-provider/src/main/resources/application.properties

```diff
@@ -154,6 +154,7 @@ fineract.loan.transactionprocessor.error-not-found-fail=${FINERACT_LOAN_TRANSACT
 # Comma separated list of loan statuses which will be recorded on change. There are two extra values: "NONE" and "ALL".
 # "NONE" disables the feature and no entries will be created, "ALL" enables the feature for all loan statuses.
 fineract.loan.status-change-history-statuses=${FINERACT_LOAN_STATUS_CHANGE_HISTORY_STATUSES:NONE}
+fineract.loan.allow-cash-and-non-cash-accrual=${FINERACT_LOAN_ALLOW_CASH_AND_NON_CASH_ACCRUAL:true}
```

Comment:
Please add 
`fineract.loan.allow-cash-and-non-cash-accrual=true` 
to `application-test.properties`